CRM/Odoosync/Upgrader.php

```diff
@@ -0,0 +1,94 @@
+<?php
+  use CRM_Odoosync_ExtensionUtil as E;
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_Odoosync_Upgrader extends CRM_Odoosync_Upgrader_Base {
+
+  /**
+   * This hook is called immediately after an extension is installed.
+   *
+   */
+  public function postInstall() {
+    $this->updateContactCustomData();
+  }
+
+  /**
+   * This hook call when extension uninstall
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  public function uninstall() {
+    $this->deleteExtensionOptionGroups();
+    $this->deleteExtensionCustomGroups();
+  }
+
+  /**
+   * Delete all option group from ext
```

Comment:
according to drupal standrads : https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards

 If a file docblock summary begins with a verb, it should be in third person singular present tense.

so in this case it should be "Deletes" instead of "Delete", please apply this for all methods.


also no need to use abbreviations here, so use extension instead of ext.

in general a more clear description would be : 

**Deletes all the option groups created by the extension**